[2024-10-16T17:58:31.892+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-16T17:58:31.901+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_cargar_cotizaciones.obtener_y_preparar_cotizaciones manual__2024-10-16T17:58:30.354161+00:00 [queued]>
[2024-10-16T17:58:31.908+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_cargar_cotizaciones.obtener_y_preparar_cotizaciones manual__2024-10-16T17:58:30.354161+00:00 [queued]>
[2024-10-16T17:58:31.909+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-16T17:58:31.917+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): obtener_y_preparar_cotizaciones> on 2024-10-16 17:58:30.354161+00:00
[2024-10-16T17:58:31.922+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2004) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-16T17:58:31.924+0000] {standard_task_runner.py:72} INFO - Started process 2009 to run task
[2024-10-16T17:58:31.924+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_cargar_cotizaciones', 'obtener_y_preparar_cotizaciones', 'manual__2024-10-16T17:58:30.354161+00:00', '--job-id', '448', '--raw', '--subdir', 'DAGS_FOLDER/dag_cargar_cotizaciones.py', '--cfg-path', '/tmp/tmp1ceqe3h1']
[2024-10-16T17:58:31.926+0000] {standard_task_runner.py:105} INFO - Job 448: Subtask obtener_y_preparar_cotizaciones
[2024-10-16T17:58:31.970+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_cargar_cotizaciones.obtener_y_preparar_cotizaciones manual__2024-10-16T17:58:30.354161+00:00 [running]> on host f03003374728
[2024-10-16T17:58:32.013+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_cargar_cotizaciones' AIRFLOW_CTX_TASK_ID='obtener_y_preparar_cotizaciones' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T17:58:30.354161+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-16T17:58:30.354161+00:00'
[2024-10-16T17:58:32.014+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-16T17:58:32.021+0000] {logging_mixin.py:190} INFO - 2024-10-15
[2024-10-16T17:58:32.021+0000] {logging_mixin.py:190} INFO - 2024-10-16
[2024-10-16T17:58:32.023+0000] {cache.py:130} INFO - Failed to create TzCache, reason: Error creating TzCache folder: '/home/***/.cache/py-yfinance' reason: [Errno 13] Permission denied: '/home/***/.cache/py-yfinance'. TzCache will not be used. Tip: You can direct cache to use a different location with 'set_tz_cache_location(mylocation)'
[2024-10-16T17:58:32.023+0000] {cache.py:336} INFO - Failed to create CookieCache, reason: Error creating CookieCache folder: '/home/***/.cache/py-yfinance' reason: [Errno 13] Permission denied: '/home/***/.cache/py-yfinance'. CookieCache will not be used. Tip: You can direct cache to use a different location with 'set_tz_cache_location(mylocation)'
[2024-10-16T17:58:34.652+0000] {logging_mixin.py:190} WARNING - [                       0%                       ]
[2024-10-16T17:58:34.685+0000] {logging_mixin.py:190} WARNING - [****************      33%                       ]  2 of 6 completed
[2024-10-16T17:58:34.697+0000] {logging_mixin.py:190} WARNING - [**********************50%                       ]  3 of 6 completed
[2024-10-16T17:58:34.704+0000] {logging_mixin.py:190} WARNING - [**********************67%*******                ]  4 of 6 completed
[2024-10-16T17:58:34.740+0000] {logging_mixin.py:190} WARNING - [**********************83%***************        ]  5 of 6 completed
[2024-10-16T17:58:34.740+0000] {logging_mixin.py:190} WARNING - [*********************100%***********************]  6 of 6 completed
[2024-10-16T17:58:34.748+0000] {warnings.py:112} WARNING - /opt/***/dags/stocks_prices_daily.py:15: FutureWarning: The previous implementation of stack is deprecated and will be removed in a future version of pandas. See the What's New notes for pandas 2.1.0 for details. Specify future_stack=True to adopt the new implementation and silence this warning.
  stacked_data = daily_stock_data.stack(level='Ticker').reset_index()

[2024-10-16T17:58:34.762+0000] {base.py:84} INFO - Retrieving connection 'redshift_conn'
[2024-10-16T17:58:36.727+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-16T17:58:36.740+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-16T17:58:36.741+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_cargar_cotizaciones, task_id=obtener_y_preparar_cotizaciones, run_id=manual__2024-10-16T17:58:30.354161+00:00, execution_date=20241016T175830, start_date=20241016T175831, end_date=20241016T175836
[2024-10-16T17:58:36.771+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-16T17:58:36.786+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-16T17:58:36.787+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
